ee/modules/650-runtime-audit-engine/images/falcosidekick/Dockerfile

```diff
@@ -1,8 +1,8 @@
 # Based on https://github.com/falcosecurity/falcosidekick/blob/41d530807f1a0294c0276e4cb42af68c8b26a659/Dockerfile
-ARG BASE_GOLANG_20_ALPINE_DEV
+ARG BASE_GOLANG_22_ALPINE_DEV
```

Comment:
Maybe we can rewrite it to werf.yaml while we still have time?

Reply:
Yes, good point.

Reply:
fixed.